packages/expo-dev-launcher/bundle/native-modules/DevMenuInternal.ts

```diff
@@ -0,0 +1,47 @@
+import { NativeModules, Platform } from 'react-native';
+
+const DevMenu = NativeModules.ExpoDevMenuInternal;
+
+export type DevMenuSettingsType = Partial<{
+  motionGestureEnabled: boolean;
+  touchGestureEnabled: boolean;
+  keyCommandsEnabled: boolean;
+  showsAtLaunch: boolean;
+}>;
+
+export async function getSettingsAsync(): Promise<DevMenuSettingsType> {
+  // TODO - native module integration
```

Comment:
is this still a TODO?

Reply:
Yes, although I'm considering just including this stuff in this PR instead of the third (draft) one I have open